docs/user-guide/compute-resources.md

```diff
@@ -91,10 +91,16 @@ runner (Docker or rkt).
 
 When using Docker:
 
-- The `spec.container[].resources.limits.cpu` is multiplied by 1024, converted to an integer, and
-  used as the value of the [`--cpu-shares`](
+- The `spec.container[].resources.requests.cpu` is converted to its millicore value,
+  multipled by 1024, and then divided by 1000, and used as the value of the [`--cpu-shares`](
   https://docs.docker.com/reference/run/#runtime-constraints-on-resources) flag to the `docker run`
```

Comment:
I wonder if it makes sense to have the documentation in terms of docker's APIs given that we have additional runtimes in the pipeline.
Instead, why not say specify using cgroups APIs?